subsys/usb/usb_device.c

```diff
@@ -499,22 +509,73 @@ static bool set_endpoint(const struct usb_ep_descriptor *ep_desc)
 
 	ep_cfg.ep_type = ep_desc->bmAttributes;
 
-	LOG_DBG("Configure endpoint 0x%x type %u MPS %u",
+	LOG_DBG("Set endpoint 0x%x type %u MPS %u",
 		ep_cfg.ep_addr, ep_cfg.ep_type, ep_cfg.ep_mps);
 
 	if (usb_dc_ep_configure(&ep_cfg) < 0) {
-		LOG_WRN("Failed to configure endpoint %x", ep_cfg.ep_addr);
+		LOG_WRN("Failed to configure endpoint 0x%02x", ep_cfg.ep_addr);
```

Comment:
I was thinking shall we use LOG_DBG() here when we have endpoint already configured.

Reply:
There is no common approach in the drivers to check if endpoint is already configured. We have that for enable and I think there is no sense to add flag and distinguish this log message in such case. This will require changing internals for each driver and will give minor enhancement.